python/servo/testing_commands.py

```diff
@@ -345,7 +345,7 @@ def test_compiletest(self, test_name=None, package=None, release=False):
                 test_patterns.append(test)
 
         if not packages:
-            packages = set(os.listdir(path.join(self.context.topdir, "tests", "compiletest")))
+            packages = set(os.listdir(path.join(self.context.topdir, "tests", "compiletest"))) - set('.DS_Store')
```

Comment:
Please revert these changes and submit them in another PR, since they're unrelated to the rest of the PR.